hbase-shell/src/main/ruby/shell.rb

```diff
@@ -333,10 +333,12 @@ def help_footer
     normalizer_switch
     normalizer_enabled
     is_in_maintenance_mode
+    clear_slow_responses
```

Comment:
Is it better to use clear_slowlog_responses?

Reply:
Using clear_slow_responses and get_slow_responses might be simpler to use? I tried following example mentioned by @apurtell on Jira: https://issues.apache.org/jira/browse/HBASE-22978

Although Java code naming refer to log, may be we can use log here also. I felt like keeping this as is. Thought?

Reply:
Is this fine with you @guangxuCheng ?

Reply:
I think it is more readable if we keep the shell command and method name consistent.     @apurtell WDYT？

Reply:
@guangxuCheng I have made changes. Please review.